.gitlab-ci.yml

```diff
@@ -23,17 +22,23 @@ cache:
 
 before_script:
   - python -V  # Print out python version for debugging
-  - pip install -r requirements.txt
-  - pip install python-coveralls
+  - pip install -r requirements.txt python-coveralls pylint pycodestyle
 
 test:
   stage: test
   script:
-    - echo $TEST_SUITE
-    - echo $TEST_SUITE > test.sh
-    - bash test.sh
-    - coverage report
-  #  - coveralls
+    # TODO bjuhasz: use tox because "setup.py test" is deprecated:
+    # https://setuptools.readthedocs.io/en/latest/setuptools.html#test-build-package-and-run-a-unittest-suite
+    - coverage run --branch --source=pyega3 -m pytest
```

Comment:
Currently, we have 58% total coverage (line 147 in the logs):
https://gitlab.ebi.ac.uk/EGA/ega-download-client/-/jobs/495650#L147